index.js

```diff
@@ -0,0 +1,55 @@
+require("dotenv").config();
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const products = require("./routes/products.router");
+const users = require("./routes/users.router");
+const carts = require("./routes/carts.router");
+const wishlists = require("./routes/wishlists.router");
+const initializeDbConnection = require("./db/db.connect");
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(cors());
+
+const port = 3000;
+
+initializeDbConnection();
+
+app.use("/products", products);
+app.use("/users", users);
+app.use("/carts", carts);
+app.use("/wishlists", wishlists);
+
+app.get("/", (req, res) => {
+  res.send("Welcome to Circlekart");
+});
+
+/**
+ * 404 Route Handler
+ * Note: Do not move. This should be the last route.
+ */
+app.use((req, res) => {
+  res.status(404).json({
+    success: false,
+    message: "route not found on server, please check.",
+  });
```

Comment:
You can extract this &  move it to a middleware folder if you want and can be called here like `app.use(routeHandler)`

Reply:
Yeah, will do that.